src/components/shared/OutlinedToggleButton.tsx

```diff
@@ -57,6 +58,7 @@ function OutlinedToggleButton({
 
     return (
         <ToggleButton
+            {...props}
```

Comment:
This is fine now but this should be at the end to allow a user to override whatever they want.